### PR TITLE
Feature/82 feedback for managers xlsx generation

### LINKED ISSUE #82: Feedback for managers xlsx generation and sending
Create a method to generate xlsx from the statistic model. The  path to save should be provided from the environment variables. Name them uniquely using the current date like `JobStat_ 2022_02_11.xlsx`.
Create a method to send the file as an email attachment for a manager.
After the file generation send the file to all the managers.

### PR DESCRIPTION
**Pull request by luxz97**
appsettings.json:
"OutputDir": "XLS_DIRECTORY_PATH"

### CODE REVIEW

**Comment by luxz97** [comment]
Both related endpoints are temporary, until scheduled xls sending is implemented, so file access/security was not a concern at this time.

**Review by tsaskater — APPROVED** [review]
looks good 🍉

File: wman-backend/Wman.Logic/Services/EmailService.cs [hunk 1]
```diff
@@ -57,6 +67,16 @@ private async Task SendEmail(string toAddress, string subject, string htmlConten
             smtpClient.EnableSsl = true;
             MailMessage mail = new MailMessage();
 
+            if (!String.IsNullOrWhiteSpace(filePath))
+            {
+                Attachment data = new Attachment(filePath, MediaTypeNames.Application.Octet);
```
Comment (pop-simon) [inline wman-backend/Wman.Logic/Services/EmailService.cs:72]: A resource is created here that is not handled after sending the email (at line 88). This causes that the Excel generation cannot be done again unless the application is restarted.
Please add the following (or something that does the same) after line 88:
```c#
if (mail.Attachments.Count > 0)
{
    foreach (var attachment in mail.Attachments)
    {
        attachment.Dispose();
    }
}
```

**Review by pop-simon — CHANGES_REQUESTED** [review]
Good job I like how you handled the Excel generation however there is a tiny issue with sending the email. 🤞

**Review by pop-simon — APPROVED** [review]

**Pull request merged**